### PR TITLE
integrating djangolib

### PR DESCRIPTION
**Pull request by domdinicola**

### CODE REVIEW

File: src/requirements/test.txt [hunk 1]
```diff
@@ -20,7 +20,6 @@ defusedxml==0.5.0
 diff-match-patch==20121119
 dj-database-url==0.5
 dj-static==0.0.6
-django-adminactions==1.5
```
Comment (dpoirier) [inline src/requirements/test.txt]: This change appears unrelated to the rest of this pull request.

File: src/requirements/local.txt [hunk 2]
```diff
@@ -54,6 +54,7 @@ djangorestframework-recursive==0.1.2
 djangorestframework-xml==1.3
 djangorestframework==3.6.4
 docutils==0.14            # via sphinx
+drf-api-checker==0.2.0
```
Comment (dpoirier) [inline src/requirements/local.txt]: Is this change related to the rest of this pull request?

Comment (domdinicola) [reply src/requirements/local.txt]: most probably the requirements were not generated correctly

File: src/requirements/test.txt (hunk 1 above)
Comment (domdinicola) [reply src/requirements/test.txt]: most probably the requirements were not generated correctly

**Comment by domdinicola** [comment]
yes @reinbach, this was the idea, right @saxix?

**Comment by domdinicola** [comment]
@reinbach @saxix @ntrncic any comments on this?
Do you like the idea to move here useful functionality?
@reinbach do you need anything here for SIR?

**Comment by domdinicola** [comment]
rest, django utils, permissions?

**Comment by reinbach** [comment]
Yes, and permissions should be merged into a complete permissions package, that handles all permissions related functionality.

File: src/etools/applications/partners/views/interventions_v2.py [hunk 3]
```diff
@@ -81,7 +82,7 @@ def get_queryset(self):
         return qs
 
 
-class InterventionListAPIView(QueryStringFilterMixin, ExportModelMixin, InterventionListBaseView):
+class InterventionListAPIView(QueryStringFilterAPIView, ExportModelMixin, InterventionListBaseView):
```
Comment (robertavram) [inline src/etools/applications/partners/views/interventions_v2.py]: can we keep as a mixin? please?
